packages/medusa/src/strategies/batch-jobs/product/export.ts

```diff
@@ -277,11 +270,15 @@ export default class ProductExportStrategy extends AbstractBatchJobStrategy<
           }
 
           products.forEach((product: Product) => {
-            const lines = this.buildProductVariantLines(product)
-            lines.forEach((line) => {
-              approximateFileSize += Buffer.from(line).byteLength
-              writeStream.write(line)
-            })
+            try {
+              const lines = this.buildProductVariantLines(product)
+              lines.forEach((line) => {
+                approximateFileSize += Buffer.from(line).byteLength
+                writeStream.write(line)
+              })
+            } catch (e) {
+              console.error(e)
+            }
```

Comment:
we should not catch the error here right? we should let it propagate

Reply:
Shit, i let it by mistake when i was doing my tests 🤣 thanks